apikana-maven-plugin/src/main/java/org/swisspush/apikana/AbstractApikanaMojo.java

```diff
@@ -158,10 +158,18 @@ protected boolean isPom() {
     protected void generatePackageJson(String version) throws IOException {
         updateJson(working("package.json"), pack -> {
             pack.put("name", mavenProject.getArtifactId());
+            pack.put("version", mavenProject.getVersion());
             final Map<String, String> scripts = (Map) pack.merge("scripts", new HashMap<>(), (oldVal, newVal) -> oldVal);
             scripts.put("apikana", "apikana");
             final Map<String, String> devDependencies = (Map) pack.merge("devDependencies", new HashMap<>(), (oldVal, newVal) -> oldVal);
+            final Map<String,List<String>> customConfig = (Map) pack.merge("customConfig", new HashMap<>(), (oldVal, newVal) -> oldVal);
+            final List<String> plugins = new ArrayList<>();
+            plugins.add("maven");
+            plugins.add("readme");
+            customConfig.put("plugins", plugins);
+            pack.put("customConfig", customConfig);
             devDependencies.put("apikana", version);
+            devDependencies.put("apikana-defaults", "0.0.0");
```

Comment:
Where is the source code for this "*apikana-defaults 0.0.0*" hosted? I'm unable to locate it.

Reply:
For organizations defining defaults in corporate registry  these can be overridden by defining apikana.defaults.version